config/environment.js

```diff
@@ -58,14 +58,14 @@ module.exports = function (environment) {
     },
 
     'ember-mapbox-composer': {
-      host: 'https://labs-layers-api.herokuapp.com',
+      host: 'http://localhost:3000',
```

Comment:
Can we change this to `localhost:3120`?  Since the default for factfinder-api is `localhost:3000`